### ISSUE TITLE
ERROR: Cannot install graph-ood and graph-ood==1.1.1 because these package versions have conflicting dependencies.

### ISSUE THREAD

**Issue by Ee1s** [divelab/GOOD#12]
Hi GOOD Team @CM-BF ,
Thanks for the great dataset. I had some problems like   #9 with the installation on Mac. 
![WeChatefaa5a21f9fafef5c370118e7506f258](https://user-images.githubusercontent.com/10851438/230101216-6d1d8fdd-fe1a-44a6-a0b4-bac56c4eb2f5.png)

I followed the solution you suggested in #9, but it didn't work.  I also tried updating typed-argument-parser==1.7.2 and that didn't work either.
Could you help me to solve this problem? Thanks.

This is my pip list:

Package                       Version
----------------------------- ---------
absl-py                       1.4.0
alabaster                     0.7.13
antlr4-python3-runtime        4.9.3
appnope                       0.1.3
asttokens                     2.2.1
attrs                         22.2.0
Babel                         2.12.1
backcall                      0.2.0
beautifulsoup4                4.12.0
cachetools                    5.3.0
captum                        0.2.0
certifi                       2022.12.7
charset-normalizer            3.1.0
cilog                         1.2.3
cloudpickle                   2.2.1
cvxopt                        1.3.0
cycler                        0.11.0
decorator                     5.1.1
dive-into-graphs              0.1.2
docutils                      0.17.1
et-xmlfile                    1.1.0
executing                     1.2.0
filelock                      3.10.7
fonttools                     4.39.3
gdown                         4.7.1
google-auth                   2.17.1
google-auth-oauthlib          0.4.6
grpcio                        1.53.0
hydra-core                    1.3.2
idna                          3.4
imagesize                     1.4.1
importlib-metadata            6.1.0
importlib-resources           5.12.0
iniconfig                     2.0.0
ipython                       8.12.0
jedi                          0.18.2
Jinja2                        3.1.2
joblib                        1.2.0
kiwisolver                    1.4.4
littleutils                   0.2.2
llvmlite                      0.39.1
Markdown                      3.4.3
MarkupSafe                    2.1.2
matplotlib                    3.5.2
matplotlib-inline             0.1.6
mpmath                        1.3.0
munch                         2.5.0
mypy-extensions               1.0.0
networkx                      2.8
numba                         0.56.4
numpy                         1.23.5
oauthlib                      3.2.2
ogb                           1.3.5
omegaconf                     2.3.0
openpyxl                      3.1.2
outdated                      0.2.2
packaging                     23.0
pandas                        2.0.0
parso                         0.8.3
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        9.5.0
pip                           23.0.1
pluggy                        1.0.0
prompt-toolkit                3.0.38
protobuf                      3.20.1
psutil                        5.9.4
ptyprocess                    0.7.0
pure-eval                     0.2.2
py                            1.11.0
pyasn1                        0.4.8
pyasn1-modules                0.2.8
Pygments                      2.14.0
pynvml                        11.4.0
pyparsing                     3.0.9
PySocks                       1.7.1
pytest                        7.1.2
python-dateutil               2.8.2
pytz                          2023.3
PyYAML                        6.0
rdkit                         2022.9.5
rdkit-pypi                    2022.9.5
requests                      2.28.2
requests-oauthlib             1.3.1
rsa                           4.9
ruamel.yaml                   0.17.21
ruamel.yaml.clib              0.2.7
scikit-learn                  1.2.2
scipy                         1.10.1
setuptools                    59.5.0
shap                          0.41.0
six                           1.16.0
slicer                        0.0.7
snowballstemmer               2.2.0
soupsieve                     2.4
Sphinx                        5.3.0
sphinx-rtd-theme              1.0.0
sphinxcontrib-applehelp       1.0.4
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        2.0.1
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.5
stack-data                    0.6.2
sympy                         1.11.1
tabulate                      0.9.0
tensorboard                   2.8.0
tensorboard-data-server       0.6.1
tensorboard-plugin-wit        1.8.1
threadpoolctl                 3.1.0
tomli                         2.0.1
torch                         1.10.1
torch-geometric               2.0.4
torch-scatter                 2.0.9
torch-sparse                  0.6.13
torchaudio                    0.10.1
torchvision                   0.11.2
tqdm                          4.64.0
traitlets                     5.9.0
typed-argument-parser         1.5.4
typing_extensions             4.5.0
typing-inspect                0.8.0
tzdata                        2023.3
urllib3                       1.26.15
wcwidth                       0.2.6
Werkzeug                      2.2.3
wheel                         0.38.4
zipp                          3.15.0

**Comment by CM-BF** [comment]
Hi Luzhi,

The problem is caused by the wrong version of [dive-into-graph](https://pypi.org/project/dive-into-graphs/). The latest `dig` version is 1.0.0. However, in your packages list, `dive-into-graphs==0.1.2` causes the conflict versions of `typed-argument-parser` 1.5.4 and 1.7.2. Therefore, the solution is to install the `dive-into-graphs==1.0.0`.

Please let me know if any questions. :smile:

**Comment by Ee1s** [comment]
Hi Shurui,
Thanks for your kind reply. I have tried to install dive-into-graphs 1.0.0, but encountered an error:

![image](https://user-images.githubusercontent.com/10851438/230530092-45673a03-f888-4a63-b689-c775a0152e76.png)

To address this issue, I attempted to install [pyscf==1.7.6.post1](https://pypi.org/project/pyscf/1.7.6.post1/#files) and got the following error:
![image](https://user-images.githubusercontent.com/10851438/230530323-f9f347cd-8b77-4990-925f-745753a22da9.png)

I am unsure if this issue can be resolved, and I am considering switching from Mac to Linux. Thank you for your assistance.

Luzhi Wang
